src/applications/disability-benefits/all-claims/tests/content/toxicExposure.unit.spec.jsx

```diff
@@ -180,7 +200,7 @@ describe('toxicExposure', () => {
         },
       };
 
-      window.sessionStorage.setItem(SHOW_TOXIC_EXPOSURE, true);
+      window.sessionStorage.setItem(SHOW_TOXIC_EXPOSURE, 'true');
```

Comment:
MIght be cleaner to add this in the `beforeEach` callback?

Reply:
for sure. i can consolidate the enabled tests into their own section